common/crypto_utils/tests/README.md

```diff
@@ -19,7 +19,9 @@ Test Build and Execution
    If the virtual environment for the current shell session is activated,
    you will the see this prompt: `(_dev)` 
 
-2. To build and run the tests run `./run_test.sh `.
+2. To build the tests run `./run_test.sh `.
+
+3. To execute the tests run `./run_test.sh test`.
 
 3. To remove generated binaries run `./run_test.sh clean `.
```

Comment:
Update to 4. Though visibly not evident on GItHub - https://github.com/hyperledger/avalon/blob/8afeab63a04b681bc44c66e807e70c27c8f1e589/common/crypto_utils/tests/README.md

Reply:
I see that now. The rendered MarkDown changed "3." to a "4." so it was not obvious.